Automattic/wp-calypso
# Jetpack: Do not request updates from site if the site is running Jetpack

Since Jetpack 4.2 we have better idea as updates sync to us what updates the sites needs. 
This will fix issues related to us hitting jetpack sites all at once. 
Which can cause problems on shared hosts. 

To test:
- Notice the updates link in Site indicator still works as expected. 
- Notice that we do not request updates from Jetpack sites by looking at the network tab in the browser console. 
- Browse around calypso and notice that nothing should be broken.
